hosts/web01/configuration.nix

```diff
@@ -0,0 +1,17 @@
+{ inputs, ... }:
+{
+  imports = [
+    inputs.disko.nixosModules.disko
+    ./hardware-configuration.nix
+    inputs.srvos.nixosModules.mixins-nginx
+    inputs.srvos.nixosModules.hardware-hetzner-cloud
+    inputs.self.nixosModules.common
+  ];
+
+  networking.hostName = "web01";
+  networking.hostId = "1cfd5aa3";
+
+  systemd.network.networks."10-uplink".networkConfig.Address = "2a01:4f9:c011:932f::/64";
```

Comment:
The cloud-init module should already assign an ipv6 address. Just tested with the CPX41 instance type.